Makefile

```diff
@@ -161,11 +161,11 @@ single-up: cleanup build manifests kustomize docker-build create-kind-cluster ki
 single-reload: build manifests kustomize docker-build kind-load-image cert-manager
 	kubectl config use-context kind-k8ssandra-0
 	$(KUSTOMIZE) build config/deployments/control-plane$(DEPLOY_TARGET) | kubectl apply --server-side --force-conflicts -f -
-	kubectl delete pod -l control-plane=k8ssandra-operator -n k8ssandra-operator
-	kubectl rollout status deployment k8ssandra-operator -n k8ssandra-operator
+	kubectl delete pod -l control-plane=k8ssandra-operator -n $(NS)
```

Comment:
I'm not sure this change was intentional?

Reply:
it was actually. The `$(NS)` version will cover cases where the deployment is done on another namespace than `k8ssandra-operator` which is the default one.

Reply:
Yeah, after our recents offline conversations about the Makefile I understood the usage of that variable, and indeed it seems to be missing from this specific target.

Reply:
Unfortunately with your changes and a non-default namespace, the target fails:

```
make NS=custom single-reload
[...]
kubectl -n custom delete pod -l control-plane=k8ssandra-operator
No resources found
kubectl -n custom rollout status deployment k8ssandra-operator
Error from server (NotFound): namespaces "custom" not found
make: *** [single-reload] Error 1
```